packages/hpe-icons/src/js/grommet/themes/base.ts

```diff
@@ -0,0 +1,40 @@
+import { light, dimension } from 'hpe-design-tokens/grommet';
+
+console.log('dimension', dimension.hpe.icon);
+console.log('light', light.hpe.color.icon);
+
+type ThemeType = {
+  global: {
+    colors: {
+      icon: (typeof light)['hpe']['color']['icon'][keyof (typeof light)['hpe']['color']['icon']];
+    };
+  };
+  icon: {
+    size: {
+      xsmall: string;
+      small: string;
+      medium: string;
+      large: string;
+      xlarge: string;
+      xxlarge: string;
+    };
+  };
+};
+
+export const base: ThemeType = {
+  global: {
+    colors: {
+      icon: light.hpe.color.icon.default,
```

Comment:
Should we include both light and dark

```suggestion
      icon: {
        light: light.hpe.color.icon.default,
        dark: dark.hpe.color.icon.default,
      },
```